doc/articles/getting-started-tutorial-1.md

```diff
@@ -0,0 +1,401 @@
+# Create a Single Page App with Uno
+
+[Download Sample](https://github.com/nventive/Uno.GettingStarted)
```

Comment:
This link didn't work for me

Reply:
that's expected, we still need to upload the repo.